dars/clearlinux/mkl/Dockerfile

```diff
@@ -34,10 +35,10 @@ RUN URL='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15816' &
 # file exists on different layers. To minimize docker
 # image size, remove the overlapped files before copy.
 RUN mkdir /os_core_install
-COPY --from=clearlinux/os-core:latest / /os_core_install/
+COPY --from=base-image / /os_core_install/
 RUN find / os_core_install | sed -e 's/os_core_install/install_root/' | xargs rm -d &> /dev/null || true
 
-FROM clearlinux/os-core:latest
+FROM base-image
```

Comment:
Did you ensure this FROM works?

Reply:
Yes, before sent this PR I built up the images and perform sanity as CI/CD of DARS